support-frontend/app/views/main.scala.html

```diff
@@ -30,6 +30,8 @@
       </script>
     }
 
+    <script defer id="stripe-js" src="https://js.stripe.com/v3/"></script>
```

Comment:
This was previously at the bottom of the body.
It prevented stripe from loading (`window.Stripe`) because the `<script>` element was not available at the point that react creates the components. Having it in the header solves this problem, and it's still marked with `defer` so I don't think it affects performance?

Reply:
still have no idea how it's possible that the entire html body hasn't been loaded and parsed by the time one is able to call 'document' within the parsed and executing javascript -- i wonder if this is something to do with the performance improvements that we did at the beginning of the year

Moving it to the header will cause it to start downloading earlier but the `defer` deprioritizes it executing. It may affect performance. Worth testing and debugging the above issue if it does.

Reply:
This was resolved by @jlieb10 [here](https://github.com/guardian/support-frontend/pull/2058/commits/c719765ee28e69232102108d96414beff32d16df)